pkgs/tools/system/runitor/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "runitor";
+  version = "0.9.2";
+  vendorSha256 = null;
+
+  src = fetchFromGitHub {
+    owner = "bdd";
+    repo = "runitor";
+    rev = "v${version}";
+    sha256 = "sha256-LuCxn4j0MlnJjSh3d18YNzNrtbqtMPxgkZttqKUGJd4";
+  };
+
+  ldflags = [
+    "-s" "-w" "-X main.Version=v${version}"
+  ];
+
+  doInstallCheck = true;
+  installCheckPhase = ''
```

Comment:
You can use `passthru.tests.version` to test the version. See https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/networking/cluster/openshift/default.nix#L55 for an example.

Reply:
[884e153](https://github.com/NixOS/nixpkgs/pull/170566/commits/884e153c0ab5be8913d92084b2fbbb94fcab7ced)